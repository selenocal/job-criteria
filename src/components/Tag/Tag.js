import React from "react";
import styled from "styled-components";
import PropTypes from "prop-types";

const Tag = ({ children }) => <StyledTag>{children}</StyledTag>;

const StyledTag = styled.span`
  display: inline-block;
  background-color: #ddd;
  font-size: 15px;
  padding: 5px 10px;
  border-radius: 4px;
  margin: 3px 3px 3px 0;
`;

Tag.propTypes = {
  children: PropTypes.string.isRequired
};

export default Tag;
