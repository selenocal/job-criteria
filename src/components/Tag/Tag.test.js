import React from "react";
import { shallow } from "enzyme";
import Tag from "./Tag";

describe("<Tag />", () => {
  it("renders Tag correctly", () => {
    const wrapper = shallow(<Tag>Test Tag</Tag>);

    expect(wrapper.contains("Test Tag")).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
