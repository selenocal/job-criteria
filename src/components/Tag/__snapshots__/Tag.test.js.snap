// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tag /> renders Tag correctly 1`] = `ShallowWrapper {}`;
